qadence/operations.py

```diff
@@ -1154,7 +1154,7 @@ def AnalogRot(
     q = _cast(QubitSupport, qubit_support)
     if isinstance(duration, str):
         duration = Parameter(duration)
-    alpha = duration * sympy.sqrt(omega**2 + delta**2) / 1000  # type: ignore [operator]
+    alpha = duration * sympy.sqrt(Parameter(omega) ** 2 + Parameter(delta) ** 2) / 1000  # type: ignore [operator]
```

Comment:
What happens here if `omega` and `delta` are already passed as `Parameter` ? Is calling `Parameter(Parameter(...))` ok?

I hadn't seen this MR earlier and also added this fix to [#244](https://github.com/pasqal-io/qadence/pull/244) but there I used the `if isinstance(..., str):` logic as done for the `duration` here.

Reply:
@jpmoutinho its fine

Reply:
Ok nice

Reply:
Nevertheless @jpmoutinho there is what we discussed in the way to test this.